core/server/pom.xml

```diff
@@ -13,14 +13,21 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.alluxio</groupId>
     <artifactId>alluxio-core</artifactId>
+    <groupId>org.alluxio</groupId>
     <version>1.5.0-SNAPSHOT</version>
   </parent>
   <artifactId>alluxio-core-server</artifactId>
-  <packaging>jar</packaging>
+  <packaging>pom</packaging>
```

Comment:
aggregating modules should use `pom` while modules that will be used as a dependency should use `jar`